cmd/serve/serve.go

```diff
@@ -288,9 +288,10 @@ func NewCommand(
 			shutdown := shutdown(db, server, eventingManager, &inventoryConsumer, log.NewHelper(logger))
 
 			if !storageOptions.DisablePersistence && consumerOptions.Enabled {
+				const maxBackoff = 30 * time.Second
```

Comment:
I see we are leveraging this in a couple places, this might actually be a good candidate to be configurable via the consumerOptions. Thoughts?

Reply:
also not a bad idea

Reply:
I think it may make sense tho as a configurable option so we could tune down the wait without having to rebuild if we needed to debug issues quickly with the consumer